layers/meta-resin-artik/recipes-support/resin-init/resin-init-board.bbappend

```diff
@@ -1 +1,10 @@
 FILESEXTRAPATHS_append_artik10 := "${THISDIR}/${PN}"
+FILESEXTRAPATHS_append_artik53x := ":${THISDIR}/${PN}"
```

Comment:
You can use here `artik530s` instead of `artik53x`

Reply:
done